readme_moodle.txt

```diff
@@ -0,0 +1,10 @@
+Description of updating ocis-php-sdk and its own requirements.
+
+1. Install composer.
+2. Set the desired version of `ocis-php-sdk` in `composer.json`.
+3. Run `composer update --no-dev`.
+4. Check the libraries are not already shipped by core. (Site Administration > Development > Third party libraries)
+   If a library is already shipped by core, add it to the `replace` section of `composer.json` and return to step 3.
+5. Update the version numbers of updated libraries in `readme_moodle.txt`.
```

Comment:
This file has these instructions, but no library version numbers.
Is that because there are no extra libraries? Only the guzzle libraries that are in the `replace` section?

Reply:
that was a mistake, the version numbers need to be adjusted in `thirdpartylibs.xml`, fixed in 0a740b517d67a72d43b6ff55211e6a50ba668d20